Documentation/Gold-Motor.md

```diff
@@ -1,21 +1,52 @@
-**New Proposed Standard Servo Motor**
+# New Standard Servo Motor
 
-We have been testing a new motor for consideration as the build standard. It has the same mounting pattern and shaft diameter.
-Gold motor  (57AIM30)
+## 57AIM30 - Gold Motor
+
+We have been testing and have decided on a new motor as the build standard. It has the same mounting pattern and shaft diameter as the old motor (iHSV57-30-XX).
+Gold Motor (57AIM30)
+
+If you wish to order a Gold Motor, you should source a 57AIM30 model that runs at 1500 RPM and has an RS485 interface (NOT the 3000 RPM or CANBUS versions)
+
+Also note that if you source your motor from somewhere other than R&D, you will need to flash the firmware with a USB to RS485 dongle before you can use it.
 
 Advantages:
-- Cheaper (20%)
-- Smaller (40% length of 180W standard)
-- Lighter (50% weight of 180W)
-- Higher peak force (30% higher in real-world test)
-- Better over-torque handling - can be configured to reduce torque output instead of disable like current 180w servo
+
+- Cheaper (20%) - dependant on availability and location.
+- Smaller (40% length of 180W standard - i.e. 60% reduction!).
+- Lighter (50% weight of 180W).
+- Higher peak force (30% higher in real-world test).
+- Better over-torque handling - can be configured to reduce torque output instead of disable like current 180w servo.
+- Is drop-in replacement for iHSV-30 series making BOM compatible.
```

Comment:
iHSV-30 should be iHSV57

Reply:
Fixed in latest commit, thanks for the spot.